conans/client/generators/cmake_common.py

```diff
@@ -363,8 +363,8 @@ class CMakeCommonMacros:
                 #     libMyLib0.dylib (compatibility version 0.0.0, current version 0.0.0)
                 #     /usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 120.0.0)
                 #     /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1197.1.1)
-                set(CMAKE_SKIP_RPATH 1)  # AVOID RPATH FOR *.dylib, ALL LIBS BETWEEN THEM AND THE EXE
-                                         # SHOULD BE ON THE LINKER RESOLVER PATH (./ IS ONE OF THEM)
+                set(CMAKE_SKIP_RPATH 1 CACHE BOOL "rpaths" FORCE)  # AVOID RPATH FOR *.dylib, ALL LIBS BETWEEN THEM AND THE EXE
```

Comment:
Unrelated change that could break. Why necessary? Move to a previous Pull request